examples/data_science/browse.py

```diff
@@ -0,0 +1,19 @@
+import os
+from pathlib import Path
+import sys
+
+from tapeagents.renderers.camera_ready_renderer import CameraReadyRenderer
+from tapeagents.tape_browser import TapeBrowser
+from tapeagents.team import TeamTape
+
+# comment this code out if loading the prompt and completions takes too long for you
+tape_dir = Path(sys.argv[1])
+exp_dir = tape_dir
+# try to find a parent directory for tape_dir path that contains tapedata.sqlite
+while not os.path.exists(exp_dir / "tapedata.sqlite") and exp_dir != Path("."):
+    exp_dir = exp_dir.parent
```

Comment:
I think if one runs the data science example all the data will just be in `tapedata.sqlite`, which is the default path, so we probably don't need this search loop. I needed it for the math branch because there SQLite DBs are scattered around a lot.